dev/bin/minikube-start.sh

```diff
@@ -48,7 +50,11 @@ if [ "${disable_extra_components}" != 'true' ]; then
   sleep 5
   kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/kube-prometheus/v0.9.0/manifests/setup/prometheus-operator-0servicemonitorCustomResourceDefinition.yaml
   . "${SCRIPT_DIR_PATH}/knative-installer.sh"
-  yes | istioctl manifest apply --set profile=default --set values.gateways.istio-ingressgateway.type="ClusterIP"
+  istioctl manifest apply --skip-confirmation --set profile=default --set values.gateways.istio-ingressgateway.type="ClusterIP"
+  sleep 5
+  # Camel-K installation
+  kubectl create namespace camel-k --dry-run=client -o yaml | kubectl apply -f -
+  kamel install --runtime-version 1.10.1 -n camel-k --force -w
```

Comment:
Side comment: I think we should avoid specifying versions directly in the scripts because it makes harder to track/update/maintain.
We do it every where so I'm fine with this PR but we should refactor the scripts to improve this (i.e. also for CI alignment)

Reply:
@danielezonca Can you create a JIRA to track that please 👍